nvtabular/workflow.py

```diff
@@ -195,6 +196,21 @@ def fit_transform(self, dataset: Dataset) -> Dataset:
         self.fit(dataset)
         return self.transform(dataset)
 
+    def fit_transform_collection(
```

Comment:
What do you think about allowing a `DatasetCollection` to be passed into the normal `fit_transform` method? Is it the additional `to_fit` arg that makes that awkward?